features/project/views/project.erb

```diff
@@ -2,54 +2,25 @@
   <header class="mdl-layout__header">
     <div class="mdl-layout__header-row">
       <span class="mdl-layout__title"><%= title %></span>
+      <div class="mdl-layout-spacer"></div>
+      <a href="/projects/<%= project.id %>/trigger" class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect mdl-button--accent">
+        Trigger Job Now
+      </a>
+    </div>
+    <div class="mdl-layout__tab-bar mdl-js-ripple-effect">
+      <a href="#scroll-tab-info" class="mdl-layout__tab is-active">Info</a>
```

Comment:
maybe `Builds` here?

Reply:
Sure! I was planning to add triggers, but i think they fit better in settings

Reply:
Oh yeah, actually that's a good point. This page could be combined info, like, here's some basic info about how the project is configured and how it's doing.

Reply:
We'll want to think about how people will most likely want to consume the information.

Reply:
Yes, i agree. This is kinda my prefactor to adding new information here. I think we can iterate fast and move things around as we please. So NBD 😸